$FreeBSD$

Hack configure to eat autogenerated qt options
and to do the main Makefile make(1) compatible.

--- configure.orig	Fri Jun 11 13:49:06 2004
+++ configure	Fri Jun 11 14:25:01 2004
@@ -82,6 +82,8 @@
 			shift
 			;;
 		--help) show_usage; exit ;;
+		--with-*)
+			shift ;;
 		*) show_usage; exit ;;
 	esac
 done
@@ -1035,14 +1037,6 @@
 	echo
 	exit 1;
 fi
-cat >Makefile.tmp <<EOT
-export QTDIR = $QTDIR
-export PATH = $QTDIR/bin:$PATH
-EOT
-cat Makefile >> Makefile.tmp
-rm -f Makefile
-cp -f Makefile.tmp Makefile
-rm -f Makefile.tmp
 
 echo
 echo Good, your configure finished.  Now run \'make\'.
